deploy/helm/distributed-compute-operator/templates/istio.yaml

```diff
@@ -14,4 +14,27 @@ spec:
   portLevelMtls:
     {{ .Values.config.webhookPort }}:
       mode: PERMISSIVE
+---
+## Disables the Istio TCP idle timeout -- required for SparkClusters
+apiVersion: networking.istio.io/v1alpha3
+kind: EnvoyFilter
```

Comment:
Doesn't this disable ALL sidecar timeouts within the namespace where it's deployed? It should ONLY target Spark clusters and nothing else.

I'm not sure this is the right place to create this resource since it doesn't provide you with programmatic access, right?